src/discretize.jl

```diff
@@ -586,87 +585,172 @@ function SciMLBase.symbolic_discretize(pde_system::PDESystem,
                                                            pde_loss_functions,
                                                            bc_loss_functions)
 
-    function full_loss_function(θ, p)
+    if bayesian
```

Comment:
Nothing in the function is reused? Make it a dispatch

Reply:
within full_loss_function reweighting and the additional loss is reused, i think i would need to write a similar function again anyways again in dispatch